components/camera/ffmpeg/ffmpeg.go

```diff
@@ -40,6 +40,18 @@ type FilterConfig struct {
 	KWArgs map[string]interface{} `json:"kw_args"`
 }
 
+// Validate ensures all parts of the config are valid.
+func (cfg *Config) Validate(path string) ([]string, error) {
+	if cfg.CameraParameters != nil {
+		if cfg.CameraParameters.Height < 0 || cfg.CameraParameters.Width < 0 {
+			return nil, fmt.Errorf(
```

Comment:
[optional] `errors.Errorf` does the same thing as `fmt.Errorf` and you won't need to import a new package

Reply:
That's part of "github.com/pkg/errors" not the standard "errors" go library (always gets me too). Not changing since that's a public archive. 

We'll move away from the archived package in static.go eventually.